src/components/AdminDashboard/adminSection.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import { PropTypes as PT } from 'prop-types';
+import { Link } from 'react-router-dom';
+
+const AdminSection = (props) => {
+  const {
+    title,
+    linkTo,
+    createLinkTo,
+    config: {
+      isLoading,
+      hasError,
+      error,
+      resultCount,
+    },
+  } = props;
+
+  return (
+    <div>
+      <h3>{title}</h3>
+      {
+        isLoading ?
```

Comment:
theres nothing on earth worse than nested ternary statements haha please do this differently

Reply:
Yeah, I know, my bad. It's been updated.